src/app/containers/workspace/content/content.controller.spec.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Created by bo on 9/12/16.
+ */
```

Comment:
Could you change this to our standard header. You can change JS template header in your IDEA to 

```
/**
 * Copyright (c) ${YEAR} The Hyve B.V.
 * This code is licensed under the GNU General Public License,
 * version 3, or (at your option) any later version.
 */
```